db/db_impl/db_impl.h

```diff
@@ -957,6 +957,13 @@ class DBImpl : public DB {
     return num_running_compactions_;
   }
 
+  // Returns the number of input iterators for currently running compactions.
+  // REQUIREMENT: mutex_ must be held when calling this function.
+  int num_running_compaction_input_iterators() {
```

Comment:
I noticed `num_running_compactions` did not follow the example of `num_running_flushes`, where `db->num_running_flushes()` gets called inside `HandleNumRunningFlushes`. `HandleNumRunningCompactions` was just accessing `num_running_compactions_` directly.